pr_stats.c

```diff
@@ -2582,3 +2582,75 @@ __print_funct_t print_avg_filesystem_stats(struct activity *a, int prev, int cur
 {
 	stub_print_filesystem_stats(a, 2, TRUE);
 }
+/*
+ ***************************************************************************
+ * Display HBA statistics. This function is used to
+ * display instantaneous and average statistics.
+ *
+ * IN:
+ * @a		Activity structure with statistics.
+ * @curr	Index in array for current sample statistics.
+ * @dispavg	TRUE if displaying average statistics.
+ * @itv		Interval of time in jiffies.
+ ***************************************************************************
+ */
+__print_funct_t stub_print_hba_stats(struct activity *a, int prev, int curr, int dispavg, unsigned long long itv)
+{
+	int i;
+	struct stats_hba *sfc,*sfp;
+
+
+	if (dis) {
+		printf("\n%-11s        HBA         txframe  txframe/s         rxframe  rxframe/s\n",
+		       (dispavg ? _("Summary:") : timestamp[curr]));
```

Comment:
This won't be a Summary but an Average once absolute frames numbers will be removed.